pkg/api/types.go

```diff
@@ -754,6 +754,10 @@ type FlexVolumeSource struct {
 	// scripts.
 	// +optional
 	SecretRef *LocalObjectReference
+	// Optional: Defaults to true. SELinuxRelabel controls relabeling of the mounted volume and allows
+	// turning this off for filesystems that don't support this functionality. e.g. fuse based filesystems
+	// +optional
+	SELinuxRelabel *bool
```

Comment:
I'm not sure I understand as I made it optional with a defaulter setting if unspecified.  
What I started with, the setting was hardwired: ```SupportsSELinux: true```
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/flexvolume/mounter-defaults.go#L50  

I only added the ability to disable it in the spec if that is what was required for whatever filesystem that FlexVolume was handling.  In our case, lizardfs+fuse.

How would I make it a no-op, and default it to true, yet provide ability to set it to false for filesystems that error out when SELinux labels are applied during the mounting procedure?  I looked over all the FlexVolume drivers, and found that all of them specified ```SupportsSELinux: {true|false}```, so I worked within what I saw as a necessary declaration.

I believe that FlexVolume is sort of special, because it can mount fuse filesystems (SupportsSELinux: false) as well as many other things like LVM logical volumes where (SupportsSELinux: true).  FlexVolume is not constrained to one setting supports all filesystems it is capable of mounting, therefore there should be some sort of user facing control that can be toggled within the API spec.